### PR TITLE
Adapted for refactor

### PR DESCRIPTION
**Pull request by eezcjkr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
     "bn.js": "^4.11.8",
     "ethereumjs-util": "^5.2.0",
     "fork-ts-checker-webpack-plugin": "^0.4.15",
-    "leap-core": "0.19.2",
+    "leap-core": "file:../parsec-lib",
```
Comment (sunify) [inline package.json]: Use `yarn link` for such stuff

File: src/components/appLayout.tsx [hunk 2]
```diff
@@ -17,59 +17,31 @@ import Message from './message';
...
   section: string;
 }
 
-@inject('tokens', 'account', 'web3', 'bridge')
+@inject('tokens', 'account', 'web3')
 @observer
 class AppLayout extends Component<AppLayoutProps, any> {
   constructor(props) {
     super(props);
-    props.bridge.address = props.bridgeAddr || props.bridge.defaultAddress;
```
Comment (sunify) [inline src/components/appLayout.tsx]: Why you've removed this feature?

Comment (eezcjkr) [reply src/components/appLayout.tsx]: There is no single bridge anymore. We would need 3 addresses for the same feature now. We might be able to replicate this feature by having a config in the UI where we can set the addresses.

File: package.json (hunk 1 above)
Comment (eezcjkr) [reply package.json]: Thanks, forgot about that.

File: src/components/appLayout.tsx (hunk 2 above)
Comment (troggy) [reply src/components/appLayout.tsx]: what's the conclusion here about removing the feature, guys?

**Pull request merged**